.travis.yml

```diff
@@ -26,16 +26,20 @@ before_script:
 
 stages:
   - name: test
-  - name: eb-deploy
-    if: (tag =~ ^v) OR (type = push)
+  - name: eb-deploy-develop
+    if: (tag =~ ^v) OR (type = push) AND (branch = develop)
+  - name: eb-deploy-staging
+    if: (tag =~ ^v) OR (type = push) AND (branch = staging)
+  - name: eb-deploy-prod
+    if: (tag =~ ^v) OR (type = push) AND (branch = prod)
 
 jobs:
   fast_finish: true
   include:
     - stage: test
       script: npm run ci:travis
       node_js: 16
```

Comment:
Not a blocker but we might want to force node 16.15.0

Reply:
will let you and @JessterB handle that @mattcgenui